assignments/calculator/solutions/Jlopezjlx/calc/calc.py

```diff
@@ -1,24 +1,40 @@
-print("CALCULADORA")
-print("OPCIONES: \n1-Suma\n2-Resta\n3-Multiplicar\n4-Dividir\n5-Restante")
+import maths
+loop = True
 
-opcion=int(input("Inserte opcion deseada: "))
-valor1 = int(input("Inserte primer valor: "))
-valor2 = int(input("Inserte segundo valor: "))
+while loop == True:
```

Comment:
Change this to:

```
while True:
```

and remove the `loop` variable. To stop the execution of the endless loop add one more operation (6-Exit) and when you get 6 as an operation stop the loop (example https://wiki.python.org/moin/WhileLoop).

Reply:
Fixed